sharktuner/sharktuner/dispatch_constraints.py

```diff
@@ -468,10 +469,14 @@ def generate_attention_vector_distribute_constraints(
             mma_intrinsics=mma_intrinsics,
             lhs_layout=pv_mma_lhs_layout,
             rhs_layout=pv_mma_rhs_layout,
-            acc_layout=None,
+            acc_layout=pv_mma_acc_layout,
         )
     ]
 
+    constraints += [
+        match_layout(qk_mma_acc_layout, pv_mma_acc_layout),
+    ]
```

Comment:
Won't this fit on a single line?
```suggestion
    constraints += [match_layout(qk_mma_acc_layout, pv_mma_acc_layout)]
```
?